app/src/main/java/org/mozilla/focus/Components.kt

```diff
@@ -24,7 +25,9 @@ class Components {
  * the <code>SessionManager</code> for now.
```

Comment:
Nit: Not related to your changes, but could we update '<code>' to '[' so that this links properly?

Reply:
Will update!